marimo/_plugins/ui/_impl/tables/narwhals_table.py

```diff
@@ -228,14 +228,16 @@ def _get_summary_internal(self, column: str) -> ColumnSummary:
                 true=cast(int, col.sum()),
                 false=cast(int, total - col.sum()),
             )
-        if col.dtype == nw.Date:
+        if (col.dtype == nw.Date) or (
+            getattr(nw, "Time", None) is not None and col.dtype == nw.Time
```

Comment:
It's _ugly_, but it does not require to bump narwhals. Let me know what you think

Reply:
this works. we can remove this once we bump